packages/grpc-js/test/test-channelz.ts

```diff
@@ -82,7 +82,7 @@ describe('Channelz', () => {
         channelzClient = new channelzGrpcObject.grpc.channelz.v1.Channelz(
           `localhost:${port}`,
           grpc.credentials.createInsecure()
-        );
+        ) as ChannelzClient;
```

Comment:
If the purpose of this change is to avoid typecasts, why do we need to add one here? If you change the typecast on line 37 to the new suggested usage, would this one still be required?

Reply:
@murgatroid99 , fixed. Thank you!